pkg/template/helper.go

```diff
@@ -60,21 +60,23 @@ func executeTestCombination(v VersionTestTemplate) error {
 func AddTestCases(names []string) ([]testCase, error) {
 	var testCases []testCase
 
-	testCase := map[string]testCase{
-		"TestDaemonset":                   testcase.TestDaemonset,
-		"TestIngress":                     testcase.TestIngress,
-		"TestDnsAccess":                   testcase.TestDnsAccess,
-		"TestServiceClusterIP":            testcase.TestServiceClusterIp,
-		"TestServiceNodePort":             testcase.TestServiceNodePort,
-		"TestLocalPathProvisionerStorage": testcase.TestLocalPathProvisionerStorage,
-		"TestServiceLoadBalancer":         testcase.TestServiceLoadBalancer,
+	tcs := map[string]testCase{
```

Comment:
Nit: Instead of `tcs` may be rename to `testCaseMap`?

Reply:
since this var is not used out of the current scope and u are creating a map, no need to add "map " in the name, in this case we can keep short and concise